build-a-nodejs-app-with-cockroachdb.md

```diff
@@ -63,7 +77,21 @@ You can copy the code or
 {% include app/txn-sample.js %}
 ~~~
 
-After running the code, use the [built-in SQL client](use-the-built-in-sql-client.html) to verify that funds were transferred from one account to another:
+Then run the code:
+
+~~~ shell
+$ node txn-sample.js
```

Comment:
same here? I don't actually feel strongly, just want to be sure this is intentional.